src/components/shared/ProjectHeader/index.tsx

```diff
@@ -83,7 +83,7 @@ export default function ProjectHeader({
               fontWeight: 500,
             }}
           >
-            {isArchived && (
+            {(isArchived || metadata?.archived) && (
```

Comment:
**suggestion**: maybe we can pass this in as part of the `isArchived` prop?

Reply:
Good suggestion. I thought about doing this but wasn't sure if it made sense since we already pass metadata prop through. Have ended up passing it through to be consistent 👍